### PR TITLE
Relativize links

### PR DESCRIPTION
**Pull request by Youssef1313**
Related to https://github.com/dotnet/docs/pull/15918#issuecomment-557148818

### CODE REVIEW

**Comment by Thraka** [comment]
@mairaw Is there something about these architecture guides that need the full links?

**Comment by mairaw** [comment]
Nope. Only for the ones that are shown on the book as URLs.

File: docs/architecture/grpc-for-wcf-developers/self-hosted.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ To learn more about querying the systemd journal from the command line with `jou
 
 When running a gRPC application in production, you should use a TLS certificate from a trusted Certificate Authority (CA). This CA could be a public CA, or an internal one for your organization.
 
-On Windows hosts, the certificate may be loaded from a secure [Certificate Store](https://docs.microsoft.com/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](https://docs.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.x509store?view=netcore-3.0). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
+On Windows hosts, the certificate may be loaded from a secure [Certificate Store](https://docs.microsoft.com/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](xref:system.security.cryptography.x509certificates.x509store). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/self-hosted.md]:
```suggestion
On Windows hosts, the certificate may be loaded from a secure [Certificate Store](/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](xref:system.security.cryptography.x509certificates.x509store). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for doing this @Youssef1313. Left a small comment.

File: docs/architecture/grpc-for-wcf-developers/self-hosted.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ To learn more about querying the systemd journal from the command line with `jou
 
 When running a gRPC application in production, you should use a TLS certificate from a trusted Certificate Authority (CA). This CA could be a public CA, or an internal one for your organization.
 
-On Windows hosts, the certificate may be loaded from a secure [Certificate Store](https://docs.microsoft.com/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](https://docs.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.x509store?view=netcore-3.0). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
+On Windows hosts, the certificate may be loaded from a secure [Certificate Store](/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](xref:system.security.cryptography.x509certificates.x509store). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/self-hosted.md]: There are still some broken links

```suggestion
On Windows hosts, the certificate may be loaded from a secure [Certificate Store](/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the <xref:System.Security.Cryptography.X509Certificates.X509Store> class. The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
```

File: docs/standard/datetime/system-text-json-support.md [hunk 3]
```diff
@@ -117,8 +117,8 @@ the converter successfully parses the data using <xref:System.DateTime.Parse(Sys
 
 If you want to write a custom <xref:System.DateTime> or <xref:System.DateTimeOffset> text representation with <xref:System.Text.Json.Utf8JsonWriter>,
 you can format your custom representation to a <xref:System.String>, `ReadOnlySpan<Byte>`, `ReadOnlySpan<Char>`, or <xref:System.Text.Json.JsonEncodedText>,
-then pass it to the corresponding [Utf8JsonWriter.WriteStringValue](https://docs.microsoft.com/dotnet/api/system.text.json.utf8jsonwriter.writestringvalue?view=netcore-3.0)
-or [Utf8JsonWriter.WriteString](https://docs.microsoft.com/dotnet/api/system.text.json.utf8jsonwriter.writestring?view=netcore-3.0) method.
+then pass it to the corresponding <xref:system.text.json.utf8jsonwriter.writestringvalue?displayProperty=nameWithType>
```
Comment (mairaw) [inline docs/standard/datetime/system-text-json-support.md]:
```suggestion
then pass it to the corresponding <xref:System.Text.Json.Utf8JsonWriter.WriteStringValue%2A?displayProperty=nameWithType>
```

File: docs/standard/datetime/system-text-json-support.md [hunk 4]
```diff
@@ -117,8 +117,8 @@ the converter successfully parses the data using <xref:System.DateTime.Parse(Sys
...
+or <xref:system.text.json.utf8jsonwriter.writestring?displayProperty=nameWithType> method.
```
Comment (mairaw) [inline docs/standard/datetime/system-text-json-support.md]:
```suggestion
or <xref:System.Text.Json.Utf8JsonWriter.WriteString%2A?displayProperty=nameWithType> method.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
There are still some broken links.

**Comment by Youssef1313** [comment]
@mairaw, changes were addressed here.

**Review by mairaw — APPROVED** [review]
Thank you @Youssef1313!

**Pull request merged**